inference.py

```diff
@@ -48,10 +49,43 @@
 
 parser.add_argument('--nosmooth', default=False, action='store_true',
 					help='Prevent smoothing face detections over a short temporal window')
+parser.add_argument('--convert_25', default=False, action='store_true',
+					help='Convert the input video to 25 fps')
 
 args = parser.parse_args()
 args.img_size = 96
 
+def check_fps(filename):
+	"""
+	Checks the fps of a video.
+	"""
+	cmd = f'ffprobe -v error -select_streams v -of default=noprint_wrappers=1:nokey=1 -show_entries stream=r_frame_rate {filename}'
```

Comment:
Does this line introduce new requirements? If so, please update the `requirements.txt` or the README with instructions. Or does ffprobe get installed along with ffmpeg as a dependency?

Reply:
Also, this line is not compatible with Python 3.5.2. Could you please change to use `.format()` or other alternatives? Thanks.